### PR TITLE
Update docfx.json

### PR DESCRIPTION
**Pull request by JoshuaPartlow**
Changing feedback to standard and removing defunct user voice url.

### CODE REVIEW

File: nodejs/docfx.json [hunk 1]
```diff
@@ -158,9 +158,7 @@
       "searchScope": [
         "Teams"
       ],
-      "feedback_system": "GitHub",
-      "feedback_github_repo": "MicrosoftDocs/msteams-docs",
```
Comment (erikadoyle) [inline nodejs/docfx.json:162]: Let's retain `feedback_github_repo` for now; need to confirm w/Teams on feedback plan

File: nodejs/docfx.json [hunk 2]
```diff
@@ -158,9 +158,7 @@
...
-      "feedback_product_url": "https://microsoftteams.uservoice.com/forums/555103-public",
+      "feedback_system": "Standard",
       "ms.date": "11/13/2017",
```
Comment (erikadoyle) [inline nodejs/docfx.json:162]: Should we update `ms.date` to reflect this change?

Comment (JoshuaPartlow) [inline nodejs/docfx.json:162]:
```suggestion
      "ms.date": "03/25/2024",
```

File: nodejs/docfx.json (hunk 1 above)
Comment (JoshuaPartlow) [reply nodejs/docfx.json:162]: Will loop back here after base update.

File: nodejs/docfx.json (hunk 2 above)
Comment (JoshuaPartlow) [reply nodejs/docfx.json]: Updated

**Pull request merged**